packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -5725,6 +5725,81 @@ Expected input body:
 }
 ```
 
+### Type_Model_Templated_TemplatedTypes_float32TemplatedType
```

Comment:
1. Do we need a subnamespace for `TemplatedTypes`?
2. It can just be `Type_Model_Templated_float32` (or `Type_Model_Templated_TemplatedTypes_float32` if we still want `TemplatedTypes` operation group